pkg/config/testdata/optimizely_config_datafile.json

```diff
@@ -260,9 +260,23 @@
       "id": "12115595439"
     }
   ],
-  "attributes": [],
+  "attributes": [
+    {
+      "id": "1042",
+      "key": "param"
+    }
+  ],
   "botFiltering": false,
   "accountId": "8272261422",
-  "events": [],
+
+  "events": [
+    {
+      "experimentIds": [
+        "42"
```

Comment:
I don't see any ID exists with 42, can you please give the proper ID which is in this datafile.

Reply:
> why .ci file is added?

I needed to trigger CI and didn't want to edit anything. I can delete this file to trigger it again.

Reply:
Yes I'll use the ID there in the file! Thanks!